setup.sh

```diff
@@ -1,15 +1,39 @@
 #!/usr/bin/env bash
 
 ###################
-# PLUGINS DIRECTORY
+# DEFINITIONS
 ###################
 
+service='Xcode'
 plugins_dir=~/Library/Developer/Xcode/Plug-ins/
 
-if [ ! -d "$plugins_dir" ]; then
-	mkdir -p $plugins_dir
+###################
+# SHUT DOWN XCODE IF IT'S RUNNING
+###################
+
+if pgrep -xq -- "${service}"; then
+  echo "Xcode is running. Attempt to shut down?"
```

Comment:
Probably add y/n "Xcode is running. Attempt to shut down? y/n"

Reply:
Updated